lib/resources/awsRegionUrls.json

```diff
@@ -3,15 +3,20 @@
   "us-east-1": "s3-website-us-east-1.amazonaws.com",
   "us-west-1": "s3-website-us-west-1.amazonaws.com",
   "us-west-2": "s3-website-us-west-2.amazonaws.com",
-  "ca-central-1": "s3-website.ca-central-1.amazonaws.com",
+  "af-south-1": "s3-website-af-south-1.amazonaws.com",
+  "ap-east-1": "s3-website-ap-east-1.amazonaws.com",
```

Comment:
should this be - `s3-website.ap-east-1.amazonaws.com`
https://docs.aws.amazon.com/general/latest/gr/s3.html